openmc/data/kalbach_mann.py

```diff
@@ -13,396 +13,154 @@
-    """Returns the emission channel energy as defined in ENDF-6 manual
-    BNL-203218-2018-INRE, Revision 215, File 6 description for LAW=1
-    and LANG=2.
-
-    Parameters
-    ----------
-    e_e: float
-        Energy of the emitted particle in the center of mass system in eV
-    awr_r: float
-        Atomic weight ratio of the residual nucleus
-    awr_e: float
-        Atomic weight ratio of the emitted particle
-
-    Returns
-    -------
-    epsilon_e: float
-        Emission channel energy in eV
-
-    """
-    epsilon_e = e_e * (awr_r + awr_e) / awr_r
-    return epsilon_e
-
-
-def _calculate_kalbach_slope(energy_projectile,
-                             energy_emitted,
-                             projectile,
-                             target,
-                             compound,
-                             emitted,
-                             residual):
-    """Calculate the Kalbach slope for projectiles other than photons
-    as defined in ENDF-6 manual BNL-203218-2018-INRE, Revision 215,
-    File 6 description for LAW=1 and LANG=2.
-
-    The entrance and emission channel energies are not calculated with
-    the AWR number, but approximated with the number of mass instead.
-
-    Parameters
-    ----------
-    energy_projectile: float
-        Energy of the projectile in the laboratory system in eV
-    energy_emitted: float
-        Energy of the emitted particle in the center of mass system in eV
-    projectile: AtomicRepresentation
-        Atomic representation of the projectile
-    target: AtomicRepresentation
-        Atomic representation of the target
-    compound: AtomicRepresentation
-        Atomic representation of the compound
-    emitted: AtomicRepresentation
-        Atomic representation of the emitted particle
-    residual: AtomicRepresentation
-        Atomic representation of the residual nucleus
-
-    Returns
-    -------
-    slope: float
-        Kalbach-Mann slope
-
-    """
-    epsilon_a = _return_entrance_channel_energy(
-        energy_projectile,
-        target.a,
-        projectile.a
-    ) / EV_PER_MEV
-    epsilon_b = _return_emission_channel_energy(
-        energy_emitted,
-        residual.a,
-        emitted.a
-    ) / EV_PER_MEV
-
-    s_a = _calculate_separation_energy(compound, target, projectile)
-    s_b = _calculate_separation_energy(compound, residual, emitted)
-
-    e_a = epsilon_a + s_a
-    e_b = epsilon_b + s_b
-
-    r_1 = min(e_a, _ET1)
-    r_3 = min(e_a, _ET3)
-
-    x_1 = r_1 * e_b / e_a
-    x_3 = r_3 * e_b / e_a
-
-    slope = _C1 * x_1 \
-        + _C2 * x_1**3 \
-        + _C3 * projectile.M * emitted.m * x_3**4
-
-    return slope
+    # Determine A, Z, and N for compound and nucleus
+    A_c = compound.a
+    Z_c = compound.z
+    N_c = compound.n
+    A_a = nucleus.a
+    Z_a = nucleus.z
+    N_a = nucleus.n
+
+    # Determine breakup energy of incident particle (ENDF-6 Formats Manual,
+    # Appendix H, Table 3)
```

Comment:
```suggestion
    # Appendix H, Table 3) in MeV
```
I would specify the unit here to clarify